mod/missions/views/default/user/mission-candidate.php

```diff
@@ -143,21 +143,22 @@
        <p class="mrgn-tp-0 mrgn-bttm-0 h4"><?php echo elgg_echo("missions:skills")?></p>
         <div>
             <?php echo $skill_set; ?>
-        </div>
-
-	</div>
-    </div>
-    <div class="user-button-content">
-        <p><a href="#mid-screen-<?php echo $user->guid;?>" aria-controls="mid-screen" class="overlay-lnk center-block" role="button">View All Skills</a></p>
+        
+        <p class=""><span class="center-block"><a href="#mid-screen-<?php echo $user->guid;?>" aria-controls="mid-screen" class="overlay-lnk" role="button">View All Skills</a></span></p>
```

Comment:
I realize it was there before, but the `aria-controls` is pointing at an ID that doesn't exist. It looks like the generated ID of the element is `mid-screen-<?php echo $user->guid;?>`